spark/src/main/scala/org/apache/spark/sql/delta/DeltaColumnMapping.scala

```diff
@@ -81,7 +81,8 @@ trait DeltaColumnMappingBase extends DeltaLogging {
 
   def isInternalField(field: StructField): Boolean =
     DELTA_INTERNAL_COLUMNS.contains(field.name.toLowerCase(Locale.ROOT)) ||
-      RowIdMetadataStructField.isRowIdColumn(field)
+      RowIdMetadataStructField.isRowIdColumn(field) ||
+      RowCommitVersion.MetadataStructField.isRowCommitVersionColumn(field)
```

Comment:
Not for now but we'll probably want to unify naming later on: `RowIdMetadataStructField` -> `RowId.MetadataStructField`